docs/content/docs/features/primary-key-table.md

```diff
@@ -158,12 +154,21 @@ Full compaction changelog producer can produce complete changelog for any type o
 
 {{< /hint >}}
 
-## Changelog Tables without Primary Keys
+## Sequence Field
 
-Changelog tables can also be used without primary keys. Users can only insert or delete a whole record from the table. No update is supported.
+By default, the primary key table determines the order according to the input order. However, in distributed computing,
+there will be some cases that lead to data disorder. At this time, you can use a time field as `sequence.field`, so
+the storage engine will use this field to sort to ensure the consistency of data sequence. For example:
```

Comment:
will use this field to determine the latest record instead. For example:

Reply:
This sentence is repeated. I delete it.